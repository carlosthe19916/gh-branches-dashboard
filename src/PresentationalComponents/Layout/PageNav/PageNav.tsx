import React from "react";
import { NavLink } from "react-router-dom";
import { Nav, NavItem, NavList, NavVariants } from "@patternfly/react-core";
import NavLinkBranches from "../../../SmartComponents/NavLinkBranches";

interface Props {}

interface State {}

export class PageNav extends React.Component<Props, State> {
  render() {
    return (
      <Nav id="nav-primary-simple" aria-label="Nav">
        <NavList variant={NavVariants.horizontal}>
          <NavItem>
            <NavLink to="/home" activeClassName="pf-m-current">
              Home
            </NavLink>
          </NavItem>
          <NavItem>
            <NavLinkBranches />
          </NavItem>
        </NavList>
      </Nav>
    );
  }
}
