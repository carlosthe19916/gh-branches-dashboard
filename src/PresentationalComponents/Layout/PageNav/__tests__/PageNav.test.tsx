import React from "react";
import { shallow } from "enzyme";
import { PageNav } from "../PageNav";

it("Test snapshot", () => {
  const wrapper = shallow(<PageNav />);
  expect(wrapper).toMatchSnapshot();
});
