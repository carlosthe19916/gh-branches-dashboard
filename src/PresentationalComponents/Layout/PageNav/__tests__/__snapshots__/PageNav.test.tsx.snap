// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot 1`] = `
<Component
  aria-label="Nav"
  id="nav-primary-simple"
>
  <NavList
    ariaLeftScroll="Scroll left"
    ariaRightScroll="Scroll right"
    className=""
    variant="horizontal"
  >
    <NavItem>
      <NavLink
        activeClassName="pf-m-current"
        to="/home"
      >
        Home
      </NavLink>
    </NavItem>
    <NavItem>
      <NavLinkBranches />
    </NavItem>
  </NavList>
</Component>
`;
